phylogenetic/defaults/auspice_config.json

```diff
@@ -31,9 +31,9 @@
   "geo_resolutions": [
     "country"
   ],
-  "maintainer": [
-    "ACEGID Redeemer's University",
-    "http://acegid.org/"
+  "maintainers": [
+    {"name": "ACEGID Redeemer's University", "url": "http://acegid.org/"},
```

Comment:
Good question, I left their name in since I did not know who to attribute the prior craft in the pipeline (ACEGID or a Nextstrain member). 

Therefore I left their name in for proper attribution for prior work (split of trees into "S" and "L" segments, discussions on the design of the pipeline, organization and which metadata fields to include, etc)

Reply:
I think fine to add attribution of prior work to the `description.md` instead of listing them as maintainers.

(Maybe a question for @trvrb who added the first auspice_config.json that included them.)

Reply:
@j23414 and I chatted about this separately and noted that the current lassa build at https://nextstrain.org/lassa/l
clarifies that ACEGID is the source of the data. We can drop them from maintainers for now.

Reply:
Dropped from maintainers, chatted with @jameshadfield to get text to be added to `description.md` later on.

Reply:
Moved from maintainers to description.md in https://github.com/nextstrain/lassa/pull/8/commits/e4d25fbaa5b701955bf421d51640d2772ceeb6b7 Please feel free to suggest different wording.